modules/bclconvert/Dockerfile

```diff
@@ -0,0 +1,10 @@
+# Dockerfile to create container with bcl-convert
+# Push to nfcore/bclconvert:<VER>
+
+FROM debian:bullseye-slim
+LABEL authors="Matthias De Smet <matthias.desmet@ugent.be>" \
+    description="Docker image containing bcl-convert"
+# Disclaimer: this container is not provided nor supported by Illumina
+
```

Comment:
I got this error when using this image in my Tower environment. `Command 'ps' required by nextflow to collect task metrics cannot be found`

```suggestion
# 'ps' command is need by some nextflow executions to collect system stats
# Install procps and clean apt cache
RUN apt-get install -y \
    procps \
    && apt-get clean -y && rm -rf /var/lib/apt/lists/*
```

Reply:
Agreed!
@ewels , could you rebuild and push the image again?